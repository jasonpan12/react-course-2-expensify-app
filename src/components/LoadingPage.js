import React from 'react';

const LoadingPage = () => (
	<div className = "loader">
		<p><img className = "loader__image" src="/images/loader.gif"/></p>
	</div>
)
export default LoadingPage;
