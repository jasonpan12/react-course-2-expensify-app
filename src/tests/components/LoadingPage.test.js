import {shallow} from "enzyme/build";
import React from "react";
import LoadingPage from "../../components/LoadingPage";

test('should render login page that matches snapshot', () => {
	const wrapper = shallow(<LoadingPage />);
	expect(wrapper).toMatchSnapshot();
});
