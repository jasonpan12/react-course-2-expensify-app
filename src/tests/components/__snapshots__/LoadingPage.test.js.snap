// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render login page that matches snapshot 1`] = `
<div
  className="loader"
>
  <p>
    <img
      className="loader__image"
      src="/images/loader.gif"
    />
  </p>
</div>
`;
